apps/frontend/src/components/review/TechnicalBulkReassignModal.tsx

```diff
@@ -0,0 +1,301 @@
+              }
+            }}
+            disableClearable
+            data-cy={`user-list-${data.instrumentId}`}
+          />
+        );
+      },
+    },
+  ];
+
+  const proposalColumns: Column<ReviewData>[] = [
+    {
+      title: 'Actions',
+      width: '10%',
+      sorting: false,
+      removable: false,
+      field: 'rowActionButtons',
+      render: RowActionButtons,
+    },
+    {
+      title: 'Proposal Id',
+      width: '15%',
+      render: (data) => data.proposal.proposalId,
+    },
+    {
+      title: 'Proposal Title',
+      render: (data) => data.proposal.title,
+    },
+    {
+      title: 'Current Reviewer',
+      width: '20%',
+      render: (data) => getFullUserName(data.review.technicalReviewAssignee),
+    },
+  ];
+
+  return (
+    <>
+      <StyledDialog
+        aria-labelledby="simple-modal-title"
+        aria-describedby="simple-modal-description"
+        open={true}
+        onClose={(): void => setReviews()}
+        maxWidth="lg"
+        fullWidth
+        title={`Bulk Reassign Proposals Techniqual reviews`}
+        data-cy="bulk-reassigne-modal"
+      >
+        <DialogContent>
+          <FormHelperText error></FormHelperText>
+          {!!proposalsWithMultipleInstrument.length && (
+            <Alert severity="warning" data-cy="multi-instrument-alert">
+              {`Warning you have selected proposal(s) with multiple ${t('instrument')}s then
+              both there reviews will apear here: ${proposalsWithMultipleInstrument.toString().replaceAll(',', ', ')}`}
+            </Alert>
+          )}
+          <MaterialTable
+            title={
+              <Typography variant="h6" component="h1">
+                Techniqual Reviews
+              </Typography>
+            }
+            columns={instrumentColumns}
+            data={reviewsSortedByInstrument.map((rev) => {
+              return {
+                ...rev,
+                reviwers: instruments
+                  ? instruments.find((inst) => inst.id === rev.instrumentId)
+                  : null,
+              };
+            })}
+            isLoading={loadingInstruments}
+            options={{
+              debounceInterval: 400,
+              idSynonym: 'instrumentId',
+              search: false,
+            }}
+            detailPanel={[
+              {
+                tooltip: 'Show Manager and Scientists',
+                render: (rowData) => {
+                  return (
+                    <Box
+                      sx={{
+                        '& tr:last-child td': {
+                          border: 'none',
+                        },
+                        '& .MuiPaper-root': {
+                          padding: '0 40px',
+                          backgroundColor: '#fafafa',
+                        },
+                      }}
+                      data-cy={`reassign-proposals-table-${rowData.rowData.instrumentId}`}
+                    >
+                      <MaterialTable
```

Comment:
Material Table needs a title, otherwise it displays the default value 
![image](https://github.com/user-attachments/assets/09b84398-8bf1-49ab-8e1b-60a1f5f9d48e)